ui/permissions_test.py

```diff
@@ -32,28 +59,95 @@ def collection():
     return factories.CollectionFactory()
 
 
-def test_is_collection_owner_anonymous_users(collection_owner_permission, collection):
+@pytest.fixture
+def moira_list():
     """
-    Test for IsCollectionOwner.has_object_permission to verify that anonymous users do not have permission
+    Returns an instance of MoiraList
+    """
+    return factories.MoiraListFactory()
+
+
+@pytest.fixture
+def video():
+    """
+    Returns an instance of Video
     """
+    return factories.VideoFactory()
+
+
+@pytest.fixture
+def request_data():
+    """
+    Fixture for tests requiring a user, request, and view
+    """
+    user = factories.UserFactory()
     request = Mock(
+        method='GET',
+        user=user
+    )
+    view = Mock(kwargs={'user': user.username})
+    return SimpleNamespace(user=user, request=request, view=view)
+
+
+@pytest.fixture
+def request_data_su():
+    """
+    Fixture for tests requiring a superuser, request, and view
+    """
+    user = factories.UserFactory()
+    user.is_superuser = True
+    user.save()
```

Comment:
Pretty sure we can just do `user = factories.UserFactory(is_superuser=True)` without the 2 lines below it